setup.py

```diff
@@ -2,7 +2,7 @@
 
 import os
 import re
-#from distutils.core import setup
+# from distutils.core import setup
```

Comment:
If we don't need this, let's get rid of it; don't need to commit commented out code.